### DISCUSSION TITLE
WordPress Menu Widget: Allow changing alignment on different screen sizes

### DISCUSSION THREAD

**Discussion by provenpaige** [elementor/elementor#22131] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I use Elementor's WordPress Menu widget for menus in the footer. A lot of times, it would look better to have the items center aligned on mobile. However, that currently requires custom CSS because the "Alignment" setting doesn't have responsive options.

### Describe the solution you'd like

I'd like to see the responsive switcher available for the "Alignment" setting.

**Comment by jrothra** [comment]
Absolutely agree! Yes! I've had sites where the aligment of the footer nav should be left on desktop and tablet, but would look much better centered on mobile. Adding the responsive option here would be very helpful.

**Reply by jrothra** [reply]
@vcpaige What custom CSS are you using?  I've tried various things and nothing is working (I'm probably overlooking the simplest, most obvious thing).

**Reply by revolterab** [reply]
I usually solve this by assigning a class to all footer menus I want centered on mobile devices, and then add a custom css:

```
@media only screen and (max-width: 767px) {
	.footermenu a {
		justify-content: center !important;
	}
}
```

The class I assigned to the menu-widget is "footermenu".
I hope this helps
